.github/workflows/main.yml

```diff
@@ -253,7 +253,7 @@ jobs:
           grep -q 'May not access' lib/beautiful/init.lua
 
       - name: Build
-        run: cd "${{ github.workspace }}/build" && make
+        run: cd "${{ github.workspace }}/build" && make -j3
```

Comment:
The Ubuntu runners only have two virtual cores. Therefore, forcing three threads makes things _slower_ because it forces the CPUs into context switching to run three threads on two cores.

Reply:
The old wisdom might be outdated now that SSDs exists. Back in the days, it was always recommended to use core+1 (unless the cores have hyperthreading) because hard drive I/O was a major factor. 3 thread on a dual core system was much faster than 2 when installing Gentoo (hours faster). It was slower if the cores already had hyper threading. Because that's exactly what hyperthreading aims to solve (bottlenecked cores).

https://web.archive.org/web/20051126053125/http://www.gentoo.org/doc/en/handbook/handbook-x86.xml?full=1

![image](https://user-images.githubusercontent.com/340384/141660318-f0951619-7b8c-4279-a535-c22c98fd52ba.png)


(ok... I am getting old)

Reply:
That's a good point, though. I didn't think of the I/O potentially being slow. And with everything being virtualized hardware in The Cloud, we can't really find out easily.

But I just realized that the build step itself only takes 10 seconds or less in CI, so there shouldn't be much of a difference between the two. So `-j3` is fine.